rsmtool/notebooks/builtin_model.ipynb

```diff
@@ -151,11 +151,13 @@
     "    ax1.set_title('Values of standardized coefficients')\n",
     "    ax1.set_xlabel('')\n",
     "    ax1.set_ylabel('')\n",
-    "    # no pie chart if we have more than 15 features or if the feature names are long (pie chart looks ugly)\n",
-    "    if len(features_used) <= 15 and longest_feature_name <= 10:\n",
+    "    # no pie chart if we have more than 15 features,\n",
+    "    # if the feature names are long (pie chart looks ugly)\n",
+    "    # or if there are any negative coefficients.\n",
+    "    if len(features_used) <= 15 and longest_feature_name <= 10 and (df_betas_sorted['relative']>=0).all():\n",
```

Comment:
Do we want a test for this?

Reply:
We don't really have tests for plots in the notebooks... It would be good in future to move them out into a separate module and write tests but that's a bigger undertaking. Not sure it's worth adding another functional test: our general `lr` test has negative coefficients while `nnlr` test has all positive coefficients.

Reply:
Sure, may be we should file an issue for adding such tests in the future?

Reply:
https://github.com/EducationalTestingService/rsmtool/issues/470